web/who-is-editing.js

```diff
@@ -14,15 +14,20 @@ $(document).ready(function(){
         });
     }
 
-    function updateWidget(action = 'editcontent'){
+    function updateWidget(action){
+        if(action === 'undefined') {
```

Comment:
Could you elaborate on what this `if` statement does?

Reply:
that was mostly sloppy